tests/flux-v2-image-automation.sh

```diff
@@ -49,14 +49,32 @@ for FILE_LOCATION in $(echo ${FILE_LOCATIONS}); do
             fi
 
             IMAGE_AUTOMATION_CHECK=$(cat imagepolicies_list.yaml  | \
-            IMAGE_POLICY_NAME="${IMAGE_POLICY}" yq eval 'select(.metadata and .kind == "ImagePolicy" and .metadata.name == env(IMAGE_POLICY_NAME) )' - | yq eval '.spec.filterTags.pattern == "^prod-[a-f0-9]+-(?P<ts>[0-9]+)"' -)
+            IMAGE_POLICY_NAME="${IMAGE_POLICY}" yq eval 'select(.metadata and .kind == "ImagePolicy" and .metadata.name == env(IMAGE_POLICY_NAME) )' - | yq eval '.spec.filterTags.pattern | "^prod-[a-f0-9]+-(?P<ts>[0-9]+)"' -)
```

Comment:
Why did you change this to `|`?
This takes the found value and pipes to regex, does that work?
The original should have worked fine as it compares the found value to the regex which is right